upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.12.yaml.template

```diff
@@ -111,10 +111,10 @@ spec:
         imagePullPolicy: IfNotPresent
         resources:
           limits:
-            memory: 170Mi
+            memory: {{ KubeDNS.MemoryLimit }}
           requests:
-            cpu: 100m
-            memory: 70Mi
+            cpu: {{ KubeDNS.CPURequest }}
+            memory: {{ KubeDNS.MemoryRequest }}
```

Comment:
Great! Would you mind updating the other file as well? 

https://github.com/kubernetes/kops/blob/189ccfe9190597b5f02395a93cff36dd8eb56722/upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.6.yaml.template#L114-L117

Reply:
Changes submitted.